src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -0,0 +1,91 @@
+import { sizes, colors, transitions } from '@/shared/theme'
+import { css } from '@emotion/react'
+import styled from '@emotion/styled/'
+
+export const Container = styled.div<CheckboxStateProps>`
+  position: relative;
+  width: max-content;
+  padding: ${sizes(2)};
+  margin: -${sizes(2)};
```

Comment:
Why do we use that negative margin?

Reply:
A negative margin, in this case, makes it so the checkbox click area is a little bit bigger since its a small component makes it easier to click